connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -263,6 +263,7 @@ object SparkConnectService extends Logging {
   private type SessionCacheKey = (String, String)
 
   private[connect] var server: Server = _
+  private[connect] var service: SparkConnectService = _
```

Comment:
actually probably not needed. This was from before I tried to just send request to the service directly, but was not satisfied, because I was not getting The Real Experience.

Reply:
hmm, If it's just for testing, it doesn't seem necessary. After removing these changes, the tests will also be successful.